pkg/function/service.go

```diff
@@ -270,14 +246,14 @@ func getValueForKey(md grpcmd.MD, key string) (string, error) {
 	keyValue := md.Get(key)
 
 	if len(keyValue) > 1 {
```

Comment:
do we need this?

Reply:
this is used to fetch the window information from the gRPC metadata, I have renamed the method.

Reply:
is this to get window information? I forget what we pass in metadata.